### DISCUSSION TITLE
About the Licence

### DISCUSSION THREAD

**Discussion by Tonychu1985** [aclap-dev/video-downloadhelper#320] [License & Purchase requests]
### What's your issue?

after buying new computer,find that the licence is not avilbale.i bought it from2019,please help

### The 10 first digits (only 10) of your license number or email address

tony.chu.tony@gmail.com,31505c6be73

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi,
I sent you an email.

**Reply by Tonychu1985** [reply]
how about my old google paid account

**Comment by Tonychu1985** [comment]
[image: image.png]

jcfrog ***@***.***> 於 2024年3月22日 週五 下午11:35寫道：

**Comment by Tonychu1985** [comment]
how about my old google paid account


朱罡蔚 ***@***.***> 於 2024年3月22日 週五 下午11:36寫道：
